public/js/grid.js

```diff
@@ -63,6 +68,34 @@ if (gridWrapper) {
       },
     },
   }).render(gridWrapper);
+
+  function deleteItem(itemId) {
+    const basePath = `${window.location.protocol}//${window.location.host}`;
+    const endpoint = `${basePath}/v1/users/${itemId}`;
```

Comment:
@lane711 We have this error, is only working on the users table, I am going to fix this later today and warn you on discord. Sorry about that!

Reply:
no worries, thx